services/IBF-pipeline/pipeline/runCron.py

```diff
@@ -6,6 +6,7 @@
 import time
 import datetime
 from settings import *
+from secrets import *
```

Comment:
@RubenGeo I get this error when running `runCron.py`

<img width="707" alt="Screenshot 2021-02-09 at 13 38 20" src="https://user-images.githubusercontent.com/3880591/107364812-3802f780-6adc-11eb-816f-b3a229d04c63.png">